documents/motions/09.2022.1 - Senior Delegates.md

```diff
@@ -41,9 +42,14 @@ Senior Delegates are WCA Staff and Full Delegates, with an additional role to ma
    2. If a new Senior Delegate is needed for an area, the WCA Board will ask for applicants among all Full Delegates.
    3. Candidates shall apply for the position within the required application period, and include all documentation that is requested in the application request.
    4. The WCA Board will decide by Majority vote which of the candidates will be appointed as new Senior Delegate(s) and for which attributed area(s) they will fulfil their role.
+   5. The WCA Board may appoint an interim Senior Delegate for a period of up to 3 months without a request for applications among all Full Delegates.
+      1. The selection of an interim Senior Delegate by the WCA Board is based on a vote in the WCA Board with at least a supermajority.
+      2. It is not possible to extend the term of an interim Senior Delegate.
+      3. The term of an interim Senior Delegate automatically ends once a Senior Delegate is appointed by the WCA Board following the orderly process outlined in section 4.1 - 4.4.
```

Comment:
If we keep the right to vote on Committee/Team Leader nominations then will need to add the line in here regarding interim Senior Delegates not getting that right.

Reply:
ah yep. Although I mainly had that provision for the team leaders so that they couldn't vote for themself / against the candidate the Board proposed through the usual process.